src/app/(home)/[not-found]/page.tsx

```diff
@@ -17,7 +19,9 @@ function NotFound() {
 					<p className={styles.infoContainer__subtitle}>Такой страницы у нас нет...</p>
 					<p className={styles.infoContainer__subtitle}>Но есть помощник финансового планирования</p>
 					<Link href={MainPath.Login} className={styles.infoContainer__navigateAction}>
-						Войти в приложение
+						<Button variant={"notification"} className={styles.button} isLarge>
+							Войти в приложение
+						</Button>
```

Comment:
can you create in src/helpers/buttonFiledValues.ts enum for variant values ​​for every button component witch contain variant field and use where necessary?